services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -659,7 +659,15 @@ module.exports = {
         maxResults,
         ...(pageToken && { pageToken }),
       };
-      return googleDataApiClient.request('youtube.playlistItems', payload);
+      try {
+        const response = await googleDataApiClient.request('youtube.playlistItems', payload);
+        return {
+          pageInfo: getAsObject(response, 'pageInfo'),
+          items: getAsArray(response, 'items').filter(video => (input.showPrivate ? video : get(video, 'status.privacyStatus') === 'public')),
+        };
+      } catch (e) {
+        return { pageInfo: {}, items: [], errors: e };
```

Comment:
This should check for the specific `e.message` (or some other identifier) to determine if the the error is indeed the one you want to catch. This ensures that other, legitimate errors are still thrown (in other words, you don't want to return this logic for _all_ errors). For example:

```js
if (e.message === 'some message you want to catch') return {};
throw e;
```

An empty object is sufficient here since the `YoutubePlaylistConnection` [resolvers](https://github.com/parameter1/base-cms/blob/c6c111e2cc386e28dc7f0c561ddb6a95864e1fe9/services/graphql-server/src/graphql/resolvers/google-data-api/youtube.js#L8) already handle the defaults.